src/Compute/Compute/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Added `vmHealh` property to ``Get-AzVm ... -Status``'s return object (VMInstanceView)
```

Comment:
VmHealth

Reply:
Perhaps rephrase to
'Added 'VmHealth' property to the return object from ``Get-AzVm`` when the parameter ``-Status`` is used.

Reply:
Resolving